test/test_valgrind.sh

```diff
@@ -79,7 +79,18 @@ for test in $(ls -1 ./umf_test-*); do
 	SUP="${WORKSPACE}/test/supp/${test}.supp"
 	OPT_SUP=""
 	[ -f ${SUP} ] && OPT_SUP="--suppressions=${SUP}"
-	HWLOC_CPUID_PATH=./cpuid valgrind $OPTION $OPT_SUP --gen-suppressions=all $test >$LOG 2>&1 || echo -n "(valgrind failed) "
+
+	# skip tests incompatible with valgrind
+	case $test in
+	./umf_test-memspace_host_all)
+		FILTER='--gtest_filter="-*allocsSpreadAcrossAllNumaNodes"'
```

Comment:
one thing to consider: do we have some functionality in these tests that still should be run under Valgrind? If so and we could split the test - we should split them.

Reply:
Yes, maybe ... It is an option to be considered.

Reply:
... but not in this PR for sure :-D

Reply:
@ldorau please add an issue about that so we don't forget to do it at our free time

Reply:
Done: https://github.com/oneapi-src/unified-memory-framework/issues/347